### PR TITLE
Update Faker Definitions to V4.1.0

### PR DESCRIPTION
**Pull request by bensw**
Please fill in this template.

- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `tsc` without errors.
~~Run `npm run lint package-name` if a `tslint.json` is present.~~ N/A

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [Faker 4.1.0](https://github.com/Marak/faker.js/releases/tag/v4.1.0)
- [x] Increase the version number in the header if appropriate.
~~If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "../tslint.json" }`.~~ N/A

### CODE REVIEW

File: faker/v3/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```
Comment (minestarks) [inline faker/v3/tsconfig.json]: ```strictNullChecks``` should be on

**Review by minestarks — CHANGES_REQUESTED** [review]

File: faker/v3/tsconfig.json (hunk 1 above)
Comment (bensw) [inline faker/v3/tsconfig.json]: This was copied from the current version. Should I also change it for the base directory?

Comment (minestarks) [inline faker/v3/tsconfig.json]: Oh I missed that. Yes please!

**Review by minestarks — APPROVED** [review]

**Comment by bensw** [comment]
@minestarks What next steps need to be taken on this review? Is there any action on my end?

**Comment by minestarks** [comment]
No action on your part. I'll give the authors @basp @Kuniwak some time to review, or merge if no comments. Thanks.

**Comment by yuit** [comment]
@bensw  thanks for the PR!

**Pull request merged**